app/routes/docs.samples.tsx

```diff
@@ -0,0 +1,213 @@
+import { Link } from '@trussworks/react-uswds'
+import dedent from 'ts-dedent'
+
+import { Highlight } from '~/components/Highlight'
+
+export default function Samples() {
+  return (
+    <>
+      <h1>Code Samples</h1>
+      <div>
+        <p>
+          We have collected a short list of functions and examples that may be
+          useful when working with GCN Notices in your code. These include
+          conversions from text/VOEvent alert formats to json and xml {'('}for
+          VOEvent alerts{')'} and examples for how to save alerts in these
+          formats.
+        </p>
+        <p>
+          For more examples, or to contribute your own ideas, see our{' '}
+          <Link href="https://github.com/nasa-gcn/utility-samples">
+            Utility Sample
+          </Link>{' '}
```

Comment:
Let's please make this repository private again. I don't want to have these code samples in _two_ places, in that repo and in this one.

Reply:
I like the idea of having a place where people can contribute their own code samples for use by others in the community.  Wouldn't that be better in a separate repo than this one?

Reply:
I don't want to be forever copying code back and forth between such an external repository and this one.

Reply:
I've thought about this more, and I think we can do a much better job of curating code samples if they are part of our documentation. I don't want to have a separate repository for them because I don't want to start maintaining a package that is a grab bag of useful but not closely related code, that people might start to rely upon as a library dependency.